Emerald.App/Emerald.App/Emerald.App.csproj

```diff
@@ -2,12 +2,13 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-	  <TargetFramework>net7.0-windows10.0.22000.0</TargetFramework>
+	  <TargetFramework>net8.0-windows10.0.22000.0</TargetFramework>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <RootNamespace>Emerald.WinUI</RootNamespace>
     <ApplicationManifest>app.manifest</ApplicationManifest>
     <Platforms>x86;x64;arm64</Platforms>
-    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+	  <RuntimeIdentifiers Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 8">win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+	  <RuntimeIdentifiers Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) &lt; 8">win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
```

Comment:
Since .NET 8 uses win-(ARCH) uses over win10-(ARCH), you can remove this condition and the right above line.

Reply:
I'll implement this suggestion now.

Reply:
oh, I just noticed that RuntimeIdentifier values has changed after .net 8. wow microsoft is trying to organize more just like me or there must a real reason for it.

Reply:
Done in #42